app/src/main/java/com/example/appmarvel/activity/CharactersListActivity.kt

```diff
@@ -34,7 +36,8 @@ class CharactersListActivity : AppCompatActivity(), KoinComponent {
     }
 
     private fun showCharactersListOnTextView(charactersList: List<Character>) {
-        binding.textViewCharactersActivityCharactersList.text = charactersList.toString()
+        binding.textViewCharactersActivityCharactersList.visibility = View.GONE
```

Comment:
this TextView is not necessary anymore as you are showing now the recycler view, please remove it

Reply:
I use the text view to show the error in case i can´t show the list of characters for any reason. However, i modified the text view and left it empty instead of the word "list", so that the text is filled whenever i get the error previously mentioned.